extensions/elasticsearch/7.x/wazuh-template.json

```diff
@@ -2713,6 +2716,37 @@
                 "type": "keyword"
               }
             }
+          },
+          "YARA": {
+            "properties": {
+              "reference": {
+                "type": "keyword"
+              },
+              "api_customer": {
+                "type": "keyword"
+              },
+              "log_type": {
+                "type": "keyword"
+              },
+              "scanned_file": {
+                "type": "keyword"
+              },
+              "rule_author": {
+                "type": "keyword"
+              },
+              "rule_name": {
+                "type": "keyword"
+              },
+              "rule_description": {
+                "type": "keyword"

```

Comment:
This field should be a text type.

Reply:
If we make it a text field it won't be possible to make aggregations and use it in a table visualization for instance.